addons/website/static/src/snippets/s_process_steps/000.scss

```diff
@@ -18,33 +60,90 @@
             .s_process_step_icon {
                 position: relative;
                 z-index: 1;
-                span:after {
-                    content: '';
-                    z-index: -1;
-                    border-top: 1px solid gray('500');
-                    @include o-position-absolute(50%, 0, 0, auto);
+                span.s_process_step_arrow {
+                    @include base_arrow;
+                    border-width: 2px;
```

Comment:
I don't see where the color and style of this border is defined, seems strange. Can you explain?